src/shared/components/post/PostFooter.tsx

```diff
@@ -68,18 +69,24 @@ function PostFooter({ id, userId }: BasePostFooterType) {
               />
             )}
           </IconButton>
-          {likeCount === 1 ? (
-            <Typography sx={{color}}>{likeCount} Like</Typography>
+          {/* {displayLikeCount === 1 ? (
+            <Typography sx={{color}}>{displayLikeCount} Like</Typography>
           ) : (
-            <Typography sx={{color}}>{likeCount} Likes</Typography>
-          )}
+            <Typography sx={{color}}>{displayLikeCount} Likes</Typography>
+          )} */}
+          <Typography sx={{ color }}>
+            {displayLikeCount} {displayLikeCount === 1 ? "Like" : "Likes"}
+          </Typography>
         </Box>
         <Box component="span" sx={{ display: "flex", alignItems: "center" }}>
-          {commentCount === 1 ? (
+          {/* {comments.length === 1 ? (
             <Typography sx={{color}}>{commentCount} Comment</Typography>
           ) : (
             <Typography sx={{color}}>{commentCount} Comments</Typography>
```

Comment:
Are you saving this for later?

Reply:
Me and Nelson were using live share and forgot to delete this. Good catch!